sktime/utils/_testing/annotation.py

```diff
@@ -1,5 +1,11 @@
 # -*- coding: utf-8 -*-
 
+__author__ = ["mloning", "fkiraly", "klam-data", "pyyim", "mgorlin"]
```

Comment:
hey all!  It might be worth checking with @fkiraly and others if you are going to add them as authors to random files.  In general in sktime we consider authorship to be somewhat of a responsibility rather than something just bestowed here and there.  (I know this because I did the same thing when I got started haha, and that is what @fkiraly told me!).

Reply:
Thanks @miraep8 , should I just remove authorship in its entirety from here? It originally started without one.

Reply:
Hmmm, I guess you can still have the author field, but can leave it empty for now.